src/WinUI.TableView.csproj

```diff
@@ -1,13 +1,27 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Uno.Sdk">
   <PropertyGroup>
-    <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
-    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
-    <WindowsSdkPackageVersion>10.0.22621.38</WindowsSdkPackageVersion>
+    <TargetFrameworks>
+      net8.0-desktop;
+      net8.0;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.26100;
+      net8.0-browserwasm;
+    </TargetFrameworks>
+    <IsPackable>true</IsPackable>
+    <UnoSingleProject>true</UnoSingleProject>
+    <OutputType>Library</OutputType>
+    <!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
     <Nullable>enable</Nullable>
-    <UseWinUI>true</UseWinUI>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DisableImplicitUnoPackages>true</DisableImplicitUnoPackages>
   </PropertyGroup>
-  
+
+  <ItemGroup>
+    <PackageReference Include="CommunityToolkit.WinUI.Behaviors" Version="8.1.240916" />
+    <PackageReference Condition="'$(TargetFramework)' == 'net8.0-windows10.0.26100'" Include="Microsoft.WindowsAppSDK" Version="1.6.250108002" />
+    <PackageReference Condition="'$(TargetFramework)' != 'net8.0-windows10.0.26100'" Include="Uno.WinUI" Version="5.5.95" />
+  </ItemGroup>
```

Comment:
This _shouldn't_™ be needed, I believe as long as we can remove the `<DisableImplicitUnoPackages>true</DisableImplicitUnoPackages>` from Jerome's comment

Reply:
correct!